### PR TITLE
feat: move font serving from cdn to static files

### LINKED ISSUE #1: Deliver font files more efficiently
Serve the font files more efficiently and faster, currently using the google fonts CDN is slowing the site.

### PR DESCRIPTION
**Pull request by tsymonowicz**
Removes links to Google Fonts CDN and moves font files to /static folder.

### CODE REVIEW

**Comment by Nintron27** [comment]
Awesome, I'll take a look at this later today.

File: src/app.css [hunk 1]
```diff
@@ -1,3 +1,19 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@font-face {
+    font-family: "Source Code Pro";
+    src: url("fonts/SourceCodePro-VariableFont_wght.ttf");
```
Comment (Nintron27) [inline src/app.css]:
```suggestion
src: url("fonts/SourceCodePro-VariableFont_wght.ttf") format("truetype");
```

File: src/app.css [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+    font-style: normal;
+    font-weight: 200 900;
+    font-stretch: 0% 200%;
+}
+
+@font-face {
+    font-family: "Source Code Pro";
+    src: url("fonts/SourceCodePro-Italic-VariableFont_wght.ttf");
```
Comment (Nintron27) [inline src/app.css]:
```suggestion
src: url("fonts/SourceCodePro-Italic-VariableFont_wght.ttf") format("truetype");
```

File: src/app.css [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
 @tailwind components;
 @tailwind utilities;
+
+@font-face {
+    font-family: "Source Code Pro";
+    src: url("fonts/SourceCodePro-VariableFont_wght.ttf");
+    font-style: normal;
+    font-weight: 200 900;
+    font-stretch: 0% 200%;
+}
```
Comment (Nintron27) [inline src/app.css:12]:
```suggestion
    font-display: swap;
}
```

File: src/app.css [hunk 4]
```diff
@@ -1,3 +1,19 @@
...
+
+@font-face {
+    font-family: "Source Code Pro";
+    src: url("fonts/SourceCodePro-Italic-VariableFont_wght.ttf");
+    font-style: italic;
+    font-weight: 200 900;
+    font-stretch: 0% 200%;
+}
```
Comment (Nintron27) [inline src/app.css:21]:
```suggestion
    font-display: swap;
}
```

**Review by Nintron27 — COMMENTED** [review]
Consider adding the font format hints, along with `font-display: swap` so fonts render immediately.

**Comment by tsymonowicz** [comment]
@Nintron27 Thanks for your feedback. Updated.

**Review by Nintron27 — APPROVED** [review]
Looks great! I'll get this merged in.

**Pull request merged**